### PR TITLE
Collections

### PR DESCRIPTION
**Pull request by teddymaulana**

### CODE REVIEW

File: src/docs/templates/collection.hbs [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+								<option value="all-accesories">All Accesories</option>
+								<option value="all">Shop All</option>
+							</select>
+						</form>
+					</div>
+					<div class="col-6 col-lg-4 d-lg-flex align-items-center justify-content-end position-relative">
+						<label class="d-none d-lg-inline-block font-weight-bold mr-lg-1">Sort By</label>
+						<form>
+							<select class="p-1 rounded mb-2 mb-lg-0 collection-grid__select custom-select">
+								<option value="Manual">Featured</option>
```
Comment (foria) [inline src/docs/templates/collection.hbs:55]: collection-grid has not css, also collection-grid__select. if don't use please remove

**Review by foria — CHANGES_REQUESTED** [review]

File: src/docs/templates/collection.hbs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+								<option value="kits-gifts">Kit & Gifts</option>
+								<option value="all-accesories">All Accesories</option>
+								<option value="all">Shop All</option>
+							</select>
+						</form>
+					</div>
+					<div class="col-6 col-lg-4 d-lg-flex align-items-center justify-content-end position-relative">
+						<label class="d-none d-lg-inline-block font-weight-bold mr-lg-1">Sort By</label>
+						<form>
+							<select class="p-1 rounded mb-2 mb-lg-0 collection-grid__select custom-select">
```
[the remark this answers is not in the document]
Comment (teddymaulana) [reply src/docs/templates/collection.hbs]: already using rizki select tag, but without form tag

File: src/docs/templates/collection.hbs (hunk 1 above)
Comment (teddymaulana) [reply src/docs/templates/collection.hbs]: I didn't remove it as now need to use it as parent of select as I added a width: auto to the select

File: src/docs/compounds/image-with-text.hbs [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	{{> html-head title="Image With Text" }}
+</head>
+
+<body>
+	<div class="container mt-4">
+		<h1>Image With Text</h1>
```
Comment (foria) [inline src/docs/compounds/image-with-text.hbs:10]: please add the reverse version too

**Review by foria — CHANGES_REQUESTED** [review]

File: src/docs/compounds/image-with-text.hbs (hunk 3 above)
Comment (teddymaulana) [reply src/docs/compounds/image-with-text.hbs:10]: already added in our story PR

File: src/docs/compounds/image-with-text.hbs [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+		<figure class="image-with-text mobile-wrapper row position-relative align-items-center mb-lg-2 mt-5">
+			<div class="col-12 col-lg-8 order-lg-last">
+				<div class="no-gutters__in-container">
+					<picture>
```
Comment (foria) [inline src/docs/compounds/image-with-text.hbs:14]: can these be merged?

File: src/docs/templates/collection.hbs [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+      <figcaption class="col-12 col-lg-4">
+        <div class="image-with-text__text-content text-left mt-lg-0 p-2 pt-lg-3 pb-lg-3 pr-lg-3 ">
+          <h1>Shop All</h1>
+          <p>Explore award-winning Bali Beauty! Shop Coco & Eve hair, body, tan and gifts. Fast worldwide delivery, unBALIevable results!</p>
+        </div>
+      </figcaption>
+    </figure>
+		<div class="row">
+			<div class="col-3">
+				<ul class="list-unstyled collection-sidebar p-3 d-none d-lg-block">
```
Comment (foria) [inline src/docs/templates/collection.hbs:28]: utility classes to hide/show should be used on the parent col element, also please use ASIDE for col-3

File: src/docs/templates/collection.hbs [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+							<option>Filter by</option>
+							<option value="tan">Tan</option>
+							<option value="hair">Hair</option>
+							<option value="body">Body</option>
+							<option value="kits-gifts">Kit & Gifts</option>
+							<option value="all-accesories">All Accesories</option>
+							<option value="all">Shop All</option>
+						</select>
+					</div>
+					<div class="col-6 col-lg-5 d-lg-flex align-items-center justify-content-end position-relative">
```
Comment (foria) [inline src/docs/templates/collection.hbs:51]: I dont think we need position-relative here

**Review by foria — CHANGES_REQUESTED** [review]

File: src/docs/compounds/image-with-text.hbs (hunk 4 above)
Comment (teddymaulana) [reply src/docs/compounds/image-with-text.hbs:14]: I have tried, but couldn't make the image to be full width without that

**Pull request merged**